server.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express');
-const app = express()
+const app = express();
```

Comment:
I doubt that this file is being used in production. Please test it on staging to make sure

Reply:
Sure, if it doesn't reflect changes in staging then I'll use `meta` tag for adding CSP.